core/domain/interaction_registry_test.py

```diff
@@ -82,7 +83,7 @@ def test_interaction_specs_json_sync_all_specs(self):
 
         spec_file = os.path.join(
             'extensions', 'interactions', 'interaction_specs.json')
-        with open(spec_file, 'r') as f:
```

Comment:
Was this correctly caught by the linter?

Reply:
Yes. I think this was caught in one of the recent commits, that's how I fixed it

Reply:
That's great! :)